src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -33,6 +37,20 @@ public void processHearingRequest(HearingWrapper wrapper) throws UnhandleableHea
         switch (wrapper.getState()) {
             case CREATE_HEARING:
                 createHearing(wrapper);
+                HearingRequestPayload hearingRequestPayload = new HearingRequestPayload();
+                HearingResponse response =
+                    hearingApi.createHearingRequest("authorisation",
+                                                    "serviceAuthorization",
+                                                    hearingRequestPayload);
+                //Fake payload for now. Ask Lucas about differences in mapping objects.
+                //check which case data to use Ask Lucas
+                wrapper.getUpdatedCaseData().setHearingID(String.valueOf(response.getHearingRequestId()));
+                //update case data method which goes to tribunals check if it's the correct method or use updateCase in UpdateCcdCaseService
```

Comment:
Use the method in CcdCaseService, although the method should really be renamed updateCaseData not updateCaseDetails in hindsight.
No method for updating the case will go to tribunals, it calls CCD directly.

Reply:
Yeah sweet and I updated it to casedata